sktime/transformations/series/signature.py

```diff
@@ -149,10 +151,10 @@ def _compute_mean_product(self, data, indices):
             return data_mean
         else:
             raise NotImplementedError("Degree higher than 3 is not implemented.")
-
+    
```

Comment:
please ensure to follow the linting guide - there should be no trailing white spaecs
https://www.sktime.net/en/latest/developer_guide/coding_standards.html